Tetris Clone C++/TetrisPiece.cpp

```diff
@@ -2,19 +2,46 @@
 #include "GridController.hpp"
 #include <iostream>
 
-const uint16_t kGRID_NONE = 0b0000000000000000;
-const uint16_t kGRID_O = 0b0000011001100000;
-const uint16_t kGRID_I = 0b0000111100000000;
-const uint16_t kGRID_L = 0b0000010001000110;
-const uint16_t kGRID_J = 0b0000001000100110;
-const uint16_t kGRID_S = 0b0000010001100010;
-const uint16_t kGRID_Z = 0b0000001001100100;
-const uint16_t kGRID_T = 0b0000001001110000;
+
```

Comment:
This segment is obscure and we have to jump through hoops to modify these shapes.
Check this out: http://tetris.wikia.com/wiki/SRS
There are tetris guidelines, we should use them.
I don't see the value added by using hexadecimal in this case. Was it merely to avoid compiler errors?
C++14 allows binary literals.